src/components/routes/Host/ListingForm/ListingHelp.tsx

```diff
@@ -104,6 +106,7 @@ const SleepingArrangement = () => (
   <div>
     <h3>Sleeping Arrangement</h3>
     <p>Sleeping Arrangement must include the quantity and size of beds.</p>
+    <p>Don’t forget to add <span>#</span> and <span>type</span> of bed!</p>
```

Comment:
Noticed a few places where `span` is used for styling. Can this be something semantic, like `em`? Given that these `span`s are styled with `${typography('emp', 5)}` emphasis appears to be the intent.

Reply:
ty. changed to `<strong>` for bold since `<em>` is italics.